meson.build

```diff
@@ -52,7 +53,7 @@ add_project_arguments(
 
 # Configuration
 
-str_check_functions = ['strdup', 'strndup']
+str_check_functions = ['strdup', 'strndup', 'sched_getaffinity']
```

Comment:
this won't work; functions in this array are cheked for existence _in the `string.h` header_